### DISCUSSION TITLE
Scope.js Lesson 20/20 javascript

### DISCUSSION THREAD

**Discussion by jasonkay843** [nodeschool/discussions#2956] [Q&A] [answered]
I'm having a hard time understand why this isn't working but doesn't pass any errors lol?

`
const a = 1; const b = 2; const c = 3;

(function firstFunction () {
    const b = 5; const c = 6;

    (function secondFunction () {
        const b = 8;
        console.log('a: ' + a + ', b: ' + b + ', c: ' + c);
        (function thirdFunction (){
            const a = 7; const c = 9;

            (function fourthFunction () {
                const a = 1; const c = 8;
                
            })
        })
    })
})
`

**Comment by jasonkay843** [comment, marked as answer]
Solved! 
console.log('a: ' + a + ', b: ' + b + ', c: ' + c); 
Removed spaces from line above and it functioned.

**Comment by mirkamolhomidov** [comment]
please mark my answer
